src/main/java/bio/terra/service/tabulardata/google/bigquery/BigQueryDatasetPdao.java

```diff
@@ -742,6 +737,122 @@ public void grantReadAccessToDataset(Dataset dataset, Collection<String> policie
         BigQueryProject.from(dataset), BigQueryPdao.prefixName(dataset.getName()), policies);
   }
 
+  // MERGE INGEST
+
+  /**
+   * This join counts the number of occurrences of each staging primary key within the target
+   * table's active records. Only primary keys which do not resolve to a single target record will
+   * be returned.
+   *
+   * @param datasetLiveViewSql SQL representing the target table's active records
+   * @return The SQL for obtaining irresolvable merge records
+   */
+  private static final String stagingRowsWithoutSingleTargetRowMatchTemplate(
```

Comment:
I'd make these not static (for consistency and if we ever need to have these interact with other service classes)

Reply:
Oh interesting...I didn't realize.  in that case, it's probably fine to keep as is